package.json

```
@@ -40,8 +40,7 @@
     "validate": "run-p lint test:ci test:e2e:headless",
     "start": "yarn storybook",
     "storybook": "storybook dev -p 6006",
-    "build-storybook": "storybook build -o dist-storybook && yarn copy-storybook-assets",
-    "copy-storybook-assets": "cp -r src/assets/ dist-storybook/assets/"
+    "build-storybook": "storybook build -o dist-storybook && yarn copy-storybook-assets"
```

Would still need to keep the `copy-storybook-assets` script so it can run as part of the `build-storybook` script:

```suggestion
    "build-storybook": "storybook build -o dist-storybook && yarn copy-storybook-assets",
    "copy-storybook-assets": "cp -r src/assets/ dist-storybook/assets/"
```